src/tasks.py

```diff
@@ -60,14 +62,23 @@ def get_task_sampler(
         "quadratic_regression": QuadraticRegression,
         "relu_2nn_regression": Relu2nnRegression,
         "decision_tree": DecisionTree,
+        "kalman_filter": KalmanFilter,
+        "noisy_kalman_filter": NoisyKalmanFilter
     }
+    print(curriculum)
     if task_name in task_names_to_classes:
         task_cls = task_names_to_classes[task_name]
         if num_tasks is not None:
             if pool_dict is not None:
                 raise ValueError("Either pool_dict or num_tasks should be None.")
-            pool_dict = task_cls.generate_pool_dict(n_dims, num_tasks, **kwargs)
-        return lambda **args: task_cls(n_dims, batch_size, pool_dict, **args, **kwargs)
+            #print(task_name == "kalman_filter")
+            if task_name == "kalman_filter":
```

Comment:
Generally case-specific checks like this are bad practice as they lead to missing otherwise obvious cases. For example, we aren't sending the curriculum argument to `NoisyKalmanFilter`. However, it seems that `curriculum` isn't used anywhere? do we need this to be passed in here?